src/nevermined/Nfts.ts

```diff
@@ -486,4 +486,30 @@ export class Nfts extends Instantiable {
             from
         )
     }
+
+    public async listOnSecondaryMarkets(
+        agreementId: string,
+        ddo: DDO,
+        assetRewards: AssetRewards,
+        numberNFTs: number,
+        provider: string,
+        owner: Account
+    ): Promise<boolean> {
+        const { nftSalesTemplate } = this.nevermined.keeper.templates
+        const providerAccounts = new Account(provider)
+        const result = await nftSalesTemplate.createAgreementFromDDO(
+            agreementId,
+            ddo,
+            assetRewards,
```

Comment:
Note to self: ## to be correct this needs to have: the marketplace split, the royalties split to the creator & the rest to the reseller